packages/amazonq/.changes/next-release/Bug Fix-d8c02c38-bdc4-492a-bf8c-6c35e35087be.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Change logic for customization override to not override if user has manually selected a customization"
```

```suggestion
	"description": "User-selected customizations not always being persisted."
```
or similar, see https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog

---

Updated